### PR TITLE
add mapper33

### PR DESCRIPTION
**Pull request by citee520**
add mapper 33, pls tell me if it' worked

### CODE REVIEW

File: src/nes/mapper/mapper033.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import {Mapper, MapperOptions} from './mapper'
+import {MirrorMode} from '../ppu/types'
+
+const kMirrorTable = [
```
Comment (tyfkda) [inline src/nes/mapper/mapper033.ts:4]: The variable `kMirrorTable` seems to be unused, so could you please remove it?

File: src/nes/mapper/mapper033.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		const kLast2Bank = maxPrg - 1
+
+		this.options.setPrgBank(0, 0)
+		this.options.setPrgBank(1, 1)
+		this.options.setPrgBank(2, kLast2Bank)
+		this.options.setPrgBank(3, kLast2Bank+1)
+	
+	
+		// Chr ROM bank
+		this.options.setWriteMemory(0x8000, 0xffff, (_adr, value) => {
```
Comment (tyfkda) [inline src/nes/mapper/mapper033.ts:29]: Could you please remove the underscore from the variable name? `adr`

File: src/nes/mapper/mapper033.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+]
+
+
+export class Mapper033 extends Mapper {
+	public static create(options: MapperOptions): Mapper {
+		return new Mapper033(options)
+	}
+	
+	constructor(options: MapperOptions) {
+		super(options, 2000)
```
Comment (tyfkda) [inline src/nes/mapper/mapper033.ts:16]: This mapper doesn't have SRAM,
so please eliminate second argument: `super(options)`

**Review by tyfkda — COMMENTED** [review]
Thank you to send the pull request!

File: src/nes/mapper/mapper033.ts (hunk 3 above)
Comment (citee520) [reply src/nes/mapper/mapper033.ts]: OK, remove under_line and SRAM(2000)

**Review by tyfkda — APPROVED** [review]
Thank you for the update!

**Pull request merged**